modules/sdk-coin-sui/src/sui.ts

```diff
@@ -23,6 +23,25 @@ export interface ExplainTransactionOptions {
   txHex: string;
 }
 
+export interface SuiParseTransactionOptions extends BaseParseTransactionOptions {
+  txHex: string;
+}
+
+interface TransactionOutput {
+  address: string;
+  amount: string;
+}
```

Comment:
is it easy to add an optional `objectId` here so callers can figure out if this is for SUI or non-SUI tokens?

if not, let's create a ticket to come back to this

Reply:
same applies to Input

Reply:
ohhh got it... lets ignore this for now then.